SwiftUIDZ/SwiftUIDZ/ClimateControlView/ClimateControlView.swift

```diff
@@ -0,0 +1,350 @@
+//
+//  ClimateControlView.swift
+//  SwiftUIDZ
+//
+//  Created by Алена Панченко on 31.01.2023.
+//
+
+import SwiftUI
+
+/// Вью - модель настроек системы кондиционирования
+final class ClimateControlViewModel: ObservableObject {
```

Comment:
```suggestion
final class ClimateControlViewModel: ObservableObject {
```
если в 1 м файле тогда вью ниже вью модель